### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by jskeet**
I know the readme.md file only refers to the Logging API rather than the Log4net provider but I'm hoping we can actually remove all the library-specific details from that file soon, so I haven't updated it further.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -300,11 +289,14 @@ See [CONTRIBUTING] for more information on how to get started.
 
 ## Versioning
 
-These libraries follow [Semantic Versioning](http://semver.org/).
+The  Google Cloud Client Libraries for .NET follow [Semantic Versioning](http://semver.org/).
```
Comment (evildour) [inline README.md]: Single space after "The"

File: README.md [hunk 2]
```diff
@@ -29,16 +29,11 @@ If you need support for other Google APIs, check out the [Google .NET API Client
...
-Most projects in this repository require [gRPC], which has a
-component written in native code. Using that component from DNX
-currently [requires a workaround][grpc-workaround]; on Windows please run `fix_dnx_grpc.bat`
-before running any gRPC code.
+which is provided as NuGet packages.
 
 ## Specifying a Project ID
 
-Most `google-cloud-dotnet` libraries require a project ID. If you don't remember yours (or haven't created a project yet), navigate to the [Google Developers Console][google-developers-console] to view your project ID (or create a new project and get the ID then). Once done, record the value and make sure to pass it as a parameter to the methods that require it.
+Most Google Cloud Libraries for .NET require a project ID. If you don't remember yours (or haven't created a project yet), navigate to the [Google Developers Console][google-developers-console] to view your project ID (or create a new project and get the ID then). Once done, record the value and make sure to pass it as a parameter to the methods that require it.
```
Comment (evildour) [inline README.md]: "or create a new project and get the ID then" might read a bit better as "or create a new project and then get the ID"

File: README.md (hunk 1 above)
Comment (jskeet) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (jskeet) [inline README.md]: Done.

**Comment by evildour** [comment]
LGTM

**Pull request merged**